.github/workflows/test.yaml

```diff
@@ -0,0 +1,9 @@
+name: test
+on: push
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run tests
+        run: ./test.sh --is-ci
```

Comment:
@nandanrao, almost matches 1:1 to `.circleci` on `fly`, https://github.com/vlab-research/fly/blob/main/.circleci/config.yml#L4-L10.